src/components/audio-player-controls/audio-player-controls.tsx

```diff
@@ -21,13 +21,44 @@ interface AudioPlayerControlsProps {
 const AudioPlayerControls = ({pluginConfig, player, onPluginsControlClick, showMorePluginsIcon}: AudioPlayerControlsProps) => {
   const playlist = useSelector((state: any) => state.engine.playlist);
   const ref = useRef<HTMLDivElement>();
+  const [selectedSpeedIndex, setSelectedSpeedIndex] = useState(1);
+  const speedValuesArray: Array<number> = [];
 
   useEffect(() => {
-    ref.current?.setAttribute('tabindex', '0');
-  }, []);
+    const element = ref.current;
+    if (!element) return;
+    element.setAttribute('tabindex', '0');
+
+    const handleKeyDown = (event: KeyboardEvent) => {
+      if (!speedValuesArray.length) return;
+
+      if (event.shiftKey && event.key === '>') {
+        event.preventDefault();
+        setSelectedSpeedIndex(prev => {
+          const newIndex = Math.min(prev + 1, speedValuesArray.length - 1);
+          player.playbackRate = speedValuesArray[newIndex];
+          return newIndex;
+        });
+      } else if (event.shiftKey && event.key === '<') {
+        event.preventDefault();
+        setSelectedSpeedIndex(prev => {
+          const newIndex = Math.max(prev - 1, 0);
+          player.playbackRate = speedValuesArray[newIndex];
+          return newIndex;
+        });
+      }
+    };
+
+    element.addEventListener('keydown', handleKeyDown);
+    return () => {
+      element.removeEventListener('keydown', handleKeyDown);
+    };
+  }, [speedValuesArray]);
 
   const _renderSpeedOptions = (playbackRates: Array<number>) => {
+    speedValuesArray.length = 0;
     return playbackRates.reduce((acc: Array<object>, speed) => {
+      speedValuesArray.push(speed);
```

Comment:
instead of keep array of speedValues you can find
`player.playbackRate` inside `player.playbackRates`, then change playbackrate based on button clicked